malaria24/ona/tasks.py

```diff
@@ -115,7 +116,9 @@ def compile_and_send_digest_email():
     cases = ReportedCase.objects.filter(digest__isnull=True)
     if not cases.exists():
         return
-
+    NationalDigest.compile_digest()
```

Comment:
Which raises another point... we need a test for this celery task, which asserts the total number of emails sent..